test/jtreg-ext/requires/VMProps.java

```diff
@@ -652,6 +655,10 @@ private String isFlagless() {
         return "" + result;
     }
 
```

Comment:
It might be useful to add a comment on `cabi()` as it may not be obvious that this is C ABI.